lib/sendIdentityValidationCode.ts

```diff
@@ -0,0 +1,48 @@
+import { FetchResult } from "apollo-link";
+import gql from "graphql-tag";
+
+import { fetchManagement } from "../src/utils/fetchManagement";
```

Comment:
Should we move this to `lib/`? or is it still used by something else in the app?

Reply:
`fetchManagement` is used in pages' getServerSideProps to retrieve identities.

Reply:
ok, so in a next PR when those move then 😁

Reply:
Do you think the lib should handle getting the identities?

Reply:
Yes, the lib should handle all relation with Connect

Reply:
and Connect Profile even